docs/radio.rst

```diff
@@ -48,8 +48,13 @@ Functions
 
 .. py:function:: on()
 
-    Turns the radio on. This needs to be explicitly called since the radio
-    draws power and takes up memory that you may otherwise need.
+    Turns the radio on.
+
+    Since MicroPython v1.1
```

Comment:
It's not really MicroPython v1.1, it's MicroPython-on-micro:bit v1.1.

Reply:
Updated in [5fae7a5](https://github.com/bbcmicrobit/micropython/compare/05c30a665f0e3f71cb93b54a96add87501ad7550..5fae7a55cb9bdef994ea31ec155503dcf140ce1a)